src/scripting.md

```diff
@@ -98,514 +211,551 @@ input: setCookie  %ORIGIN% foo=bar
 Navigates the browser to the provided URL and waits for it to complete.
 Browser Support: IE, Chrome, Firefox, Safari
 
 ```markup
-usage: navigate	<url>
-example: navigate	http://webmail.aol.com
+usage:   navigate <url>
+example: navigate http://webmail.aol.com
 
 <url> - URL to provide the browser for navigation (same as you would enter into the address bar)
 ```
 
-#### click
-Triggers a click event for the identified DOM element. This version does not have an implied wait and the script will continue running after the event is submitted (see clickAndWait for the wait version).
+#### `click`
+Triggers a click event for the identified DOM element. This version does not have an implied wait and the script will continue running after the event is submitted (see [`clickAndWait`](#clickandwait) for the wait version).
 Browser Support: IE, Chrome, Firefox
 ```markup
-usage: click	<attribute=value>
-example: click	title=Delete (del)
+usage:   click <wpt-selector>
+example: click title=Delete
 
-<attribute=value> - DOM element to click on
+<wpt-selector> - DOM element to click on
 ```
 
-#### clickAndWait
+For the list of supported selectors, see [Selectors](#selectors).
+
+#### `clickAndWait`
 Triggers a click event for the identified DOM element and subsequently waits for browser activity to complete.
 Browser Support: IE, Chrome, Firefox
 ```markup
-usage: clickAndWait	<attribute=value>
-example: clickAndWait	innerText=Send
+usage:   clickAndWait <wpt-selector>
+example: clickAndWait innerText=Send
 
-<attribute=value> - DOM element to click on
+<wpt-selector> - DOM element to click on
 ```
 
-#### selectValue
+For the list of supported selectors, see [Selectors](#selectors).
+
+#### `selectValue`
 Selects a value from a dropdown list of the given DOM element.
 Browser Support: IE
 ```markup
-usage: selectValue	<attribute=value>	<value>
-example: selectValue	id=country	usa
+usage:   selectValue <wpt-selector> <value>
+example: selectValue id=country usa
 
-<attribute=value> - DOM element to select the value of
+<wpt-selector> - DOM element to select the value of
 <value> - value to use
 ```
 
-#### sendClick / sendClickAndWait
-Creates a javascript OnClick event and sends it to the indicated element.
+For the list of supported selectors, see [Selectors](#selectors).
+
+#### `sendClick` / `sendClickAndWait`
+Creates a JavaScript `onclick` event and sends it to the indicated element.
 Browser Support: IE
 ```markup
-usage: sendClickAndWait	<attribute=value>
-example: sendClickAndWait	innerText=Send
+usage:   sendClickAndWait <wpt-selector>
+example: sendClickAndWait innerText=Send
 
-<attribute=value> - DOM element to send the click event to
+<wpt-selector> - DOM element to send the click event to
 ```
 
-#### type (AndWait)
+For the list of supported selectors, see [Selectors](#selectors).
+
+For the difference between `sendClick` and `sendClickAndWait`, see [`click`](#click) and [`clickAndWait`](#clickandwait).
+
+#### `type` / `typeAndWait`
```

Comment:
Note: this replaces “(AndWait)” with the full command name (“typeAndWait”). People may be ⌘F-ing for `typeAndWait`, and they won’t find the command if its name is never mentioned in full!

Reply:
👍